src/Console/Command/HelpCommand.php

```diff
@@ -143,6 +143,15 @@ public static function getHelpCopy()
 
     <info>$ PHP_CS_FIXER_FUTURE_MODE=1 php %command.full_name% -v --diff</info>
 
+Rules
+-----
+
+Use the following command to quickly understand what a rule will do to your code:
+
+.. code-block:: bash
```

Comment:
This is added automatically so it doesn't need to be added here (this results in this line being duplicated in the actual README).